src/index.html

```diff
@@ -17,6 +17,9 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="minute-hand minute-hand--speed-up"></div>
```

Comment:
**In addition to the basic functionality create a BEM modifier called speed-up for your stopwatch block:**


`speed-up` for your stopwatch block, not for minute-hand and second-hand